src/ert/cli/main.py

```diff
@@ -49,8 +49,8 @@ def run_cli(args: Namespace, _: Any = None) -> None:
     # Create logger inside function to make sure all handlers have been added to
     # the root-logger.
     logger = logging.getLogger(__name__)
-    for job in ert_config.forward_model_list:
-        logger.info("Config contains forward model job %s", job.name)
+    for step in ert_config.forward_model_steps:
+        logger.info("Config contains forward model job %s", step.name)
```

Comment:
Not changing the logged string deliberate?

Reply:
Yeah, was not sure if it was a good idea wrt changing logs behavior